week-3/D-array-filter/exercise.js

```diff
@@ -6,12 +6,20 @@
   - Do not edit any of the existing code
 */
 
-var pairsByIndexRaw = [[0, 3], [1, 2], [2, 1], null, [1], false, "whoops"];
+ var pairsByIndexRaw = [[0, 3], [1, 2], [2, 1], null, [1], false, "whoops"];
+ var pairsByID = [];
+ 
+ pairsByIndexRaw.filter(function(element) {
+   if (element !== null && element !==false && element !== "whoops" && element.length > 1) {
```

Comment:
This works, but what if the input contains other arbitrary values like "hello" or "oops". Remember the "typeof" keyword can be handy in this situation!